src/Workspaces/CSharp/Portable/Indentation/CSharpIndentationService.cs

```diff
@@ -35,10 +31,10 @@ protected override AbstractFormattingRule GetSpecializedIndentationFormattingRul
         }
 
         protected override AbstractIndenter GetIndenter(
-            ISyntaxFactsService syntaxFacts, SyntaxTree syntaxTree, TextLine lineToBeIndented, IEnumerable<AbstractFormattingRule> formattingRules, OptionSet optionSet, CancellationToken cancellationToken)
+            Document document, SyntaxTree syntaxTree, TextLine lineToBeIndented, IEnumerable<AbstractFormattingRule> formattingRules, OptionSet optionSet, CancellationToken cancellationToken)
         {
             return new Indenter(
-                syntaxFacts, syntaxTree, formattingRules,
+                document, syntaxTree, formattingRules,
```

Comment:
this indentation service and indenter separation seems unnecessary.

Reply:
I agree.  I can try to remove the split in a followup PR as i continue to clean htis all up.